src/app/StudentView.tsx

```diff
@@ -95,13 +103,9 @@ const AddGradeOverlay: React.FunctionComponent<{ studentID: number, refreshTrans
 
                         <FormControl mt={4} isRequired>
                             <FormLabel>Grade</FormLabel>
-                            <NumberInput defaultValue={95} name="grade">
-                                <NumberInputField/>
-                                <NumberInputStepper>
-                                    <NumberIncrementStepper/>
-                                    <NumberDecrementStepper/>
-                                </NumberInputStepper>
-                            </NumberInput>
+                            <Select placeholder="Select option">
+                            {gradeList.map(val => (<option value={val}>{val}</option>))}
+                            </Select>
```

Comment:
I agree with using the chakra's Select component here since it works well for selecting a single option from a large set of pre-defined options and prevents users from making invalid input.